dev/VSIX/Directory.Build.targets

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+    <!--
+        This target is used to temporarily update the package version we use in the VSIX manifests.
```

Comment:
What does 'temporarily' mean? What's the expiration data on that?

Reply:
for the duration of the build, until RestorePackageVersion reverts all back.  Fleshed out the comment to that effect.